pkg/reconciler/layer2.go

```diff
@@ -81,79 +150,38 @@ func (r *reconcile) reconcileLayer2(l2vnis []networkv1alpha1.Layer2NetworkConfig
 		})
 	}
 
-	existing, err := r.netlinkManager.ListL2()
-	if err != nil {
-		return err
-	}
+	return desired, nil
+}
 
-	delete := []nl.Layer2Information{}
-	for _, cfg := range existing {
+func createToDelete(existing, desired []nl.Layer2Information) []nl.Layer2Information {
```

Comment:
Same here, the function name isn't great. Maybe something like `determineToBeDeleted()` or `findToBeDeleted()` instead?

Reply:
Done. Went with `determineToBeDeleted`.